### PR TITLE
Use `nevermore` instead of ad-hoc local crate

### PR DESCRIPTION
**Pull request by JarvisCraft**
# Description

This replaces the usage of ad-hoc local `from_never` crate with the external `nevermore`.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -37,14 +37,14 @@ document-features = "0.2"
 tracing = "0.1"
 tracing-subscriber = "0.3"
 
-# Macro utils
+# Utilities
+nevermore = "0.1"
```
Comment (foaw) [inline Cargo.toml]: Should be `0.1.1`, taken the zeros out of the equation.

**Review by foaw — DISMISSED** [review]

File: Cargo.toml (hunk 1 above)
Comment (JarvisCraft) [reply Cargo.toml]: Fair enough, will fix it tomorrow

File: Cargo.toml [hunk 2]
```diff
@@ -37,14 +37,14 @@ document-features = "0.2"
 tracing = "0.1"
 tracing-subscriber = "0.3"
 
-# Macro utils
+# Utilities
```
Comment (golf25565) [inline Cargo.toml:38]: Move the `thiserror` dependency over here too

**Review by golf25565 — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 2 above)
Comment (JarvisCraft) [reply Cargo.toml:38]: Done

File: Cargo.toml (hunk 1 above)
Comment (JarvisCraft) [reply Cargo.toml]: Done

**Review by golf25565 — APPROVED** [review]

**Pull request merged**